pkgs/by-name/hp/hpack/package.nix

```diff
@@ -0,0 +1,16 @@
+{
+  lib,
+  haskell,
+  haskellPackages,
+}:
+let
+  hpack =
+    if
+      builtins.compareVersions haskellPackages.hpack.version haskellPackages.hpack_0_37_0.version < 0
+    then
+      haskellPackages.hpack_0_37_0
+    else
+      lib.warn "`hpack` is newer than `hpack_0_37_0`. Please remove the override in Nixpkgs: ${builtins.toPath ./package.nix}" haskellPackages.hpack;
```

Comment:
I've added a warning here in case `hpack` gets updated past `0.37.0` without us noticing.